tf/modules/ecs/ec2_capacity_provider/launch_template.tf

```diff
@@ -0,0 +1,82 @@
+resource "aws_launch_template" "launch_template" {
+  name                   = "${var.name}_launch_template"
+  instance_type          = var.instance_type
+  image_id               = var.ami_id == null ? data.aws_ami.ecs_optimized.id : var.ami_id
+  vpc_security_group_ids = var.security_group_ids
+  user_data              = base64encode(local.user_data)
+  update_default_version = true
+  key_name               = var.key_name
+
+  ebs_optimized = var.ebs_size_gb > 0
+
+  dynamic "block_device_mappings" {
+    for_each = var.ebs_size_gb > 0 ? [{}] : []
+
+    content {
+      // The instance volume used by Docker
+      device_name = "/dev/xvdcz"
```

Comment:
Is this a guarantee made by AWS? Seems odd to use a hardcoded device name instead of a UUID

Reply:
Yeah, it currently is for ecs-optimised AMIs (https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-ami-storage-config.html#ecs-AMI-LVM). It assumes that ecs-optimised AMI is used though so I might clarify that in readme or something.